python_src/src/lamp_py/migrations/env.py

```diff
@@ -0,0 +1,96 @@
+from logging.config import fileConfig
+
+from alembic import context
+
+from lamp_py.postgres.postgres_utils import get_local_engine
+from lamp_py.runtime_utils.import_env import load_environment
+
+# load local environment, if running alembic locally
+load_environment()
+
+# this is the Alembic Config object, which provides
+# access to the values within the .ini file in use.
+config = context.config
+
+# gate to make sure alembic is run using -n flag
+if config.config_ini_section == "alembic":
+    raise SyntaxError("Run alembic with -n flag to specifiy Database name.")
+
+# get database name from -n flag when alembic is run from cmd line
+db_name = config.config_ini_section
+
+# Interpret the config file for Python logging.
+# This line sets up loggers basically.
+if config.config_file_name is not None:
+    fileConfig(config.config_file_name)
+
+# add your model's MetaData object here
+# for 'autogenerate' support
+# from myapp import mymodel
+# target_metadata = mymodel.Base.metadata
+from lamp_py.postgres.postgres_schema import SqlBase
+
+# using dictionary for engine and target_metadata to support migrating multiple dbs
+# each dictionary name should have a section defined in alembic.ini that
+# matches the key used in the db_details dictionary
+db_details = {
+    "performance_manager": {
+        "engine": get_local_engine(),
+        "target_metadata": SqlBase.metadata,
+    }
+}
+
+# other values from the config, defined by the needs of env.py,
+# can be acquired:
+# my_important_option = config.get_main_option("my_important_option")
+# ... etc.
+
+
+# def run_migrations_offline() -> None:
+#     """Run migrations in 'offline' mode.
+
+#     This configures the context with just a URL
+#     and not an Engine, though an Engine is acceptable
+#     here as well.  By skipping the Engine creation
+#     we don't even need a DBAPI to be available.
+
+#     Calls to context.execute() here emit the given string to the
+#     script output.
+
+#     """
+#     url = config.get_main_option("sqlalchemy.url")
+#     context.configure(
+#         url=url,
+#         target_metadata=target_metadata,
+#         literal_binds=True,
+#         dialect_opts={"paramstyle": "named"},
+#     )
+
+#     with context.begin_transaction():
+#         context.run_migrations()
```

Comment:
if we aren't going to use this, i think we should remove it.

if we are, it could use a comment for the context of where we will.